### DISCUSSION TITLE
使用uni-app框架开发多端小程序时如何使用?

### DISCUSSION THREAD

**Discussion by MQL9011** [galacean/discussion#52] [Mini Program（小程序）]
手上项目需要同时支持 微信小程序和支付宝小程序.
所以采用了uni-app开发框架.
但是uni-app中的canvas不兼容oasis, 希望能得到解决, 万分感谢

**Comment by eyworldwide** [comment]
问题重复：https://github.com/ant-galaxy/oasis-engine/issues/1320
